pkg/kubelet/types/labels.go

```diff
@@ -21,6 +21,7 @@ const (
 	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
 	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
 	KubernetesContainerNameLabel = "io.kubernetes.container.name"
+	ContainerdKindLabel = "io.cri-containerd.kind"
```

Comment:
gofmt? I think test will fail because of it.

Reply:
My fault,THX!  Done